UPGRADE.md

```diff
@@ -2,6 +2,11 @@
 
 ## dev-develop
 
+## Search
+
+- Adapter namen changed e.g. from `massive_search_adapter.<adaptername>` to just `<adaptername>` in
+configuration.
```

Comment:
@dantleech I think you are also missing the changed xml tags/attributes from the massive search bundle here. Is there also a PR for sulu-standard, which corrects the dist files?